examples/AccessControl/AccessControl.ino

```diff
@@ -460,7 +460,7 @@ uint8_t findIDSLOT( byte find[] ) {
 ///////////////////////////////////////// Find ID From EEPROM   ///////////////////////////////////
 boolean findID( byte find[] ) {
   uint8_t count = EEPROM.read(0);     // Read the first Byte of EEPROM that
```

Comment:
I guess you want to fix `count` has a value of `255` (`0xff`).
I did not created this example but I suggest, that the value `255` is not a valid id. the function should abort here.